osh/hub/other/settings_util.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+import os
+
+
+def get_secret(name, directory):
+    try:
+        with open(os.path.join(directory, name)) as f:
```

Comment:
`encoding="utf-8"` ?

Reply:
Thank you for the suggestion but `utf-8` is the default on all reasonably modern Unix-like systems.  The only major OS which still defaults to something else is Windows which is unsupported by OSH anyway.

Reply:
Yes. From the buildins : `In text mode, if encoding is not specified the encoding used is platform dependent: locale.getpreferredencoding(False) is called to get the current locale encoding.`
```
[root@osh-test-worker-001 ~]# python
Python 3.9.18 (main, Jan  4 2024, 00:00:00) 
[GCC 11.4.1 20230605 (Red Hat 11.4.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import locale
>>> print(locale.getpreferredencoding(False))
UTF-8
```